### PR TITLE
Update TableSetup

### PR DESCRIPTION
**Pull request by emmanuelsteven**
In this PR,
- I created a PostgreSQL Database called vet clinic.
- I created a table for animals.
- I populated the table with the required data.
-  I queried the data inserted.
ScreenShots ;
Screenshot 2
Screenshot 3
update screenshot.
Screenshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,15 +63,15 @@ After you're finished please remove all the comments and instructions!
...
-**[your_project__name]** is a...
+**[Vet Clinic]** is a SQL database project has a table called "animals" with columns for Id, name, date of birth, escape attempts, neutered status, and weight in kilograms.This table provides a structured format to store and manage information about animals.The Vet Clinic tries  efficiently track and handle animal-related data within the veterinary clinic's operations.
 
-## 🛠 Built With <a name="built-with"></a>
+## 🛠 Built With <a name="built-with">PostgreSQL</a>
 
-### Tech Stack <a name="tech-stack"></a>
+### Tech Stack <a name="tech-stack">BackEnd</a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (mckent05) [inline README.md]: - Please remove all placeholders from your readme, they were only put in place to give info on details to fill the section with, please fix this as it'd help make your readme more professional 🙏

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+VALUES ('Agumon', date '2020-02-03', 0, true, '10.23');
+
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES ('Gabumon', date '2018-11-15', 2, true, '8');
+
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES ('Pikachu', date '2021-01-07', 0, false, '15.04');
+
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES ('De', date '2017-05-12', 2, true, '11');
```
Comment (mckent05) [inline data.sql]: - You have inputted the wrong information for this data, please fix this, according to the project requirements it should be:
```sql
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Devimon', '2017/05/12', '5', 'true', '11.0');
```

File: data.sql [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES ('Agumon', date '2020-02-03', 0, true, '10.23');
+
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES ('Gabumon', date '2018-11-15', 2, true, '8');
+
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES ('Pikachu', date '2021-01-07', 0, false, '15.04');
```
Comment (mckent05) [inline data.sql]: - You have inputted the wrong information for this data, please fix this, according to the project requirements it should be:
```sql
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Pikachu', '2021/01/07', '1', 'false', '15.04');
```

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @emmanuelsteven, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job creating your animals table 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @emmanuelsteven 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- All changes requested have been implemented 🥇  
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**